Clarify First Item page (#161)
FabricMC/fabric-docs: develop/items/first-item.md

This will not work if you've marked the item as damageable, as the stack size is always 1 for damageable items to prevent duplication exploits.
:::

> For example, you can create a new "Suspicious Substance" by adding this static field to your ModItems class.

Reviewer: ```suggestion
In this example, we will create a "Suspicious Substance" item, by adding this static field to your `ModItems` class.
```

@[code transcludeWith=:::2](@/reference/latest/src/main/java/com/example/docs/item/ModItems.java)

However, when you go in-game, you can see that our item doesn't exist! This is because there are no references to the ModItem class which would initialize it when Minecraft starts.

To force initialization of the static field, you can add a public static initialize method to your class and call it from your `ModInitializer` class. Currently, this method doesn't need anything inside it.

> ModItems.java

Reviewer: If you want to label these code blocks, use h4 headings.

@[code transcludeWith=:::3](@/reference/latest/src/main/java/com/example/docs/item/ModItems.java)

Your class that implements "ModInitializer"

@[code transcludeWith=:::1](@/reference/latest/src/main/java/com/example/docs/item/FabricDocsReferenceItems.java)

Calling a method on a class statically initializes it if it hasn't been previously loaded - this means that all `static` fields are evaluated. This is what this dummy `initialize` method is for.

> With that added, running your mod will now allow you to /give yourself your {mod-id}:suspicious_substance.

Reviewer: Wrap templates in \`\` please. Eg; `{mod-id}: suspicious_substance.`

You could alternatively have:

````md
Once you've added this method, you can run your mod and use the following command to give you the suspicious substance item;

```mcfunction
/give @s <mod-id>:suspicious_substance
```

Make sure to replace `<mod-id>` with your mod's ID!
````

> But it still won't show up when you press 'E', for that, you will need to register the item to an Item Group, described next.

Reviewer: Be more specific;

It wont show up within the creative inventory...

Keybinds aren't fixed, especially vanilla ones like `Open Inventory`
